bin/adhoc-scripts/parseUnifiedCampaigns.py

```diff
@@ -276,14 +311,24 @@ def main():
             key = proxy
         hdict = {'cert': cert, 'key': key, 'pycurl': True}
         mgr = ReqMgrAux(opts.url, hdict, logger=logger)
-    if opts.dburi:
+    if opts.cin:
+        # then simply use this json file where campaign configurations are
+        # already in the WMCore format
+        with open(opts.cin) as jo:
```

Comment:
If you add test_campaigns, then you should reuse opts.fin here. And, don't use 2 letters for code variables. Python style always suggest to use 3 or more. Better use `istream` which is much clear then `jo`.